examples/react/virtualized-rows/src/fixedHeightTable.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  Row,
+  SortingState,
+  useReactTable,
+} from '@tanstack/react-table'
+import { useVirtualizer } from '@tanstack/react-virtual'
+
+import { Person } from './makeData'
+
+type FixedHeightTableProps = {
+  // The data to render
+  data: any
+  // The columns to render
+  columns: ColumnDef<Person>[]
+  // The height of the table
+  // @default 500px
+  height?: number
+}
+
+/**
+ * Renders fixed height virtualised table
+ */
+export function FixedHeightTable({
+  data,
+  columns,
+  height = 500,
+}: FixedHeightTableProps) {
+  const [sorting, setSorting] = React.useState<SortingState>([])
+
+  const table = useReactTable({
+    data,
+    columns,
+    state: {
+      sorting,
+    },
+    onSortingChange: setSorting,
+    getCoreRowModel: getCoreRowModel(),
+    getSortedRowModel: getSortedRowModel(),
+    debugTable: true,
+  })
+
+  const tableContainerRef = React.useRef<HTMLDivElement>(null)
+
+  const { rows } = table.getRowModel()
+  const rowVirtualizer = useVirtualizer({
+    count: rows.length,
+    getScrollElement: () => tableContainerRef.current,
+    estimateSize: () => 54,
+    overscan: 10,
```

Comment:
Let's we need to pass correct keys to virtualizer it's important when rows change position
```suggestion
    overscan: 10,
    getItemKey: React.useCallback(index => rows[index].id, [rows])
```

Reply:
Added, but had to update callback to get rid of type errors